consensus/polybft/wallet/key.go

```diff
@@ -35,3 +40,31 @@ func (k *Key) Sign(b []byte) ([]byte, error) {
 
 	return s.Marshal()
 }
+
+func (k *Key) SignEcdsa(b []byte) ([]byte, error) {
+	return k.raw.Ecdsa.Sign(b)
+}
```

Comment:
We could have inlined this function, since it is used only in the `SignEcdsaMessage`.

Reply:
https://github.com/0xPolygon/polygon-edge/pull/865/commits/e9f82c8aa934344e1b727726a10daa473becae37